### PR TITLE
[Bug] Fix ppl commands

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description

Piped functions in ppl were not running

### Issues Resolved

https://app.asana.com/0/1208080319527398/1208216692717508/f

## Screenshot

image

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by ashwin-pc — DISMISSED** [review]
Sorry approved in haste

**Review by ashwin-pc — COMMENTED** [review]
Why are we doing this? We should not be modifying the query on the client side. Why do we need to do this?

File: src/plugins/query_enhancements/public/search/ppl_search_interceptor.ts [hunk 1]
```diff
@@ -73,8 +73,10 @@ export class PPLSearchInterceptor extends SearchInterceptor {
     const query: Query = this.queryService.queryString.getQuery();
     const dataset = query.dataset;
     if (!dataset || !dataset.timeFieldName) return query;
+    const [baseQuery, ...afterPipeParts] = query.query.split('|');
+    const afterPipe = afterPipeParts.length > 0 ? ` | ${afterPipeParts.join('|').trim()}` : '';
     const timeFilter = this.getTimeFilter(dataset.timeFieldName);
-    return { ...query, query: query.query + timeFilter };
+    return { ...query, query: baseQuery + timeFilter + afterPipe };
```
Comment (joshuali925) [inline src/plugins/query_enhancements/public/search/ppl_search_interceptor.ts:79]: would be good if this logic has tests, maybe later as this class doesn't have unit tests yet

Comment (ashwin-pc) [reply src/plugins/query_enhancements/public/search/ppl_search_interceptor.ts:79]: +1

**Comment by sumukhswamy** [comment]
Dont have permissions to merge this in

**Pull request merged**